src/tapis-ui/_common/Collapse/Collapse.tsx

```diff
@@ -17,29 +18,41 @@ const Collapse: React.FC<CollapseProperties> = ({
   open,
   className,
   children,
+  isCollapsable,
```

Comment:
You can do:

```
isCollapsable=true
```

to define a default true value. then you can get rid of line #28